frontend/src/i18n/locales/de/frequent.json

```
@@ -0,0 +1,124 @@
+  "Appbar Tools": "Appbar-Tools",
+  "show more": "mehr anzeigen",
+  "Version": "Version",
+  "Git Commit": "Git Commit",
+  "Try Again": "Erneut versuchen",
+  "Back": "Zurück",
+  "Next": "Weiter",
+  "Finish": "Beenden",
+  "Offline": "Offline",
+  "No": "Nein",
+  "Yes": "Ja",
+  "Toggle fullscreen": "Vollbild ein/aus",
+  "Close": "Schließen",
+  "Find": "finden",
+  "All namespaces": "Alle Namespaces",
+  "Type": "Typ",
+  "Reason": "Ereignis",
+  "From": "Von",
+  "Message": "Nachricht",
+  "Age": "Alter",
+  "Skip": "Überspringen",
+  "Read more": "Mehr lesen",
+  "Dismiss": "Ablehnen",
+  "Create / Apply": "Erstellen / Anwenden",
+  "Deleting item {{ itemName }}…": "Lösche Element {{ itemName }} …",
+  "Cancelled deletion of {{ itemName }}.": "Löschen von {{ itemName }} abgebrochen.",
+  "Deleted item {{ itemName }}.": "Element {{ itemName }} gelöscht.",
+  "Error deleting item {{ itemName }}.": "Fehler beim Löschen des Elements {{ itemName }}.",
+  "Edit": "Bearbeiten",
+  "Editor": "Editor",
+  "Documentation": "Dokumentation",
+  "Undo": "Rückgängig machen",
+  "Are you sure?": "Sind Sie sicher?",
+  "Undo Changes": "Änderungen rückgängig machen",
+  "Save & Apply": "Speichern & Anwenden",
+  "Loading resource data": "Lade Ressourcendaten",
+  "Mount Path": "Mount Path",
+  "from": "von",
+  "I/O": "I/O",
+  "Ready": "Bereit",
+  "Waiting": "Warten",
+  "Running": "Läuft",
+  "Error": "Error",
+  "Restart Count": "Anzahl der Neustarts",
+  "Restarting {{ itemName }}…": "Starte {{ itemName }}…",
+  "Cancelled restarting {{ itemName }}.": "Der Neustart von {{ itemName }} wurde abgebrochen.",
+  "Restarted {{ itemName }}.": "Neugestartet {{ itemName }}.",
+  "Failed to restart {{ itemName }}.": "Neustart von {{ itemName }} fehlgeschlagen.",
+  "Restart": "Neu starten",
+  "restart": "Neustart",
+  "Are you sure you want to restart {{ name }}?": "Sind Sie sicher, dass Sie {{ name }} neu starten wollen?",
+  "Scaling {{ itemName }}…": "Skaliere {{ itemName }}…",
+  "Cancelled scaling {{ itemName }}.": "Skalierung von {{ itemName }} abgebrochen.",
+  "Scaled {{ itemName }}.": "{{ itemName }} skaliert .",
+  "Failed to scale {{ itemName }}.": "Skalierung von {{ itemName }} fehlgeschlagen.",
+  "Scale": "Skalieren",
+  "scale": "Skala",
+  "Scale Replicas": "Scale-Replikas",
+  "Decrement": "Dekrementieren",
+  "Increment": "Inkrementieren",
+  "A large number of replicas may negatively impact the cluster's performance": "Eine große Anzahl von Replikas kann sich negativ auf die Leistung des Clusters auswirken",
+  "Show filter": "Filter anzeigen",
+  "Search": "Suchen",
+  "Filter": "Filter",
+  "Clear": "Löschen",
+  "sort up": "nach oben",
+  "sort down": "nach unten",
+  "sort swap": "tauschen",
+  "Refresh": "Aktualisieren",
+  "Add": "Hinzufügen",
+  "Data": "Daten",
+  "Group": "Gruppe",
+  "Versions": "Versionen",
+  "Full name": "Vollständiger Name",
+  "Spawn": "erzeugen",
```

```suggestion
  "Spawn": "Erzeugen",
```
Capitalised to match everything else.